libs/designsystem/src/lib/components/text-link/text-link.component.spec.ts

```diff
@@ -0,0 +1,175 @@
+import { SpectatorHost, createHostFactory } from '@ngneat/spectator';
+import { RouterTestingModule } from '@angular/router/testing';
+import { MockComponent } from 'ng-mocks';
+
+import { TextLinkComponent } from './text-link.component';
+import { DesignTokenHelper } from '../../helpers';
+import { SizeDirective } from '../../directives';
+import { IconComponent } from '../icon';
+
+const fontSize = DesignTokenHelper.fontSize;
+const lineHeight = DesignTokenHelper.lineHeight;
+const size = DesignTokenHelper.size;
+
+const getColor = DesignTokenHelper.getColor;
+
+describe('TextLinkComponent', () => {
+  let spectator: SpectatorHost<TextLinkComponent>;
+
+  const createHost = createHostFactory({
+    component: TextLinkComponent,
+    declarations: [MockComponent(IconComponent), SizeDirective],
+    imports: [RouterTestingModule],
+  });
+  describe(`text link should be `, () => {
```

Comment:
Don't the trailing space is necessary:
```suggestion
  describe(`text link should be`, () => {
```